docs/release/trg-4/trg-4-5.md

```diff
@@ -0,0 +1,26 @@
+---
+title: TRG 4.05 - Container registries
+---
+
+| Author               | Status | Created      | Post-History    |
+|----------------------|--------|--------------|-----------------|
+| Catena-X System Team | Draft  | 14-Sept-2022 | n/a             |
+| Catena-X System Team | Active | 05-Jan-2023  | Initial release |
+
+## Description
+
+In Eclipse Tractusx we are using one central container registry.
+
+Currently this central registry is the [GitHub Packages feature](https://github.com/features/packages), the GitHub organization catenax-ng contains all [our packages](https://github.com/orgs/catenax-ng/packages).
```

Comment:
Can you please update the links to tractus-x

Reply:
Sorry, I was wrong here. Catenax-ng is indeed the only registry we use to publish images for demonstration purposes